pydm/tests/test_data/alarm_stylesheet.css

```diff
@@ -0,0 +1,120 @@
+PyDMWidget[alarmSeverity="0"]{}
```

Comment:
There's some stuff we can do to make this a little bit less verbose, like specifying a whole bunch of selectors and giving them all the same style.

Reply:
Done.